data-prepper-plugins/aws-lambda/src/main/java/org/opensearch/dataprepper/plugins/lambda/processor/ResponseCardinality.java

```diff
@@ -0,0 +1,40 @@
+package org.opensearch.dataprepper.plugins.lambda.processor;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum ResponseCardinality {
+    STRICT("strict"),
+    AGGREGATE("aggregate");
+
+    private final String value;
+
+    ResponseCardinality(String value) {
+        this.value = value;
+    }
+
+    @JsonValue
+    public String getValue() {
+        return value;
+    }
+
+    private static final Map<String, ResponseCardinality> RESPONSE_CARDINALITY_MAP = new HashMap<>();
+
+    static {
+        for (ResponseCardinality type : ResponseCardinality.values()) {
+            RESPONSE_CARDINALITY_MAP.put(type.getValue().toLowerCase(), type);
+        }
+    }
+
+    @JsonCreator
+    public static ResponseCardinality fromString(String value) {
+        if (value == null) {
```

Comment:
Do not return a default value for `null` here. This can result in odd behavior such as the following being allowed.

```
response_cardinality: null
```